File: IM&RTC/RTC/MLVB/DEMO/CustomCaptureRender/Version_1/app/src/main/java/com/project/mlvbv2demo/MainActivity.java
package com.project.mlvbv2demo;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.app.ActivityCompat;

import android.Manifest;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;


import com.project.mlvbv2demo.activity.CustomActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        requestPermissions(new String[]{Manifest.permission.CAMERA,  Manifest.permission.SYSTEM_ALERT_WINDOW,Manifest.permission.RECORD_AUDIO
                ,  Manifest.permission.READ_PHONE_STATE});
    }

    /**
     * 获取多个权限
     *
     * @param permissions 获取权限的数组
     */
    private void requestPermissions(String[] permissions) {
        ActivityCompat.requestPermissions(this, permissions, 1);
    }

    public void btn(View view) {
        startActivity(new Intent(this, CustomActivity.class));
    }
}

File: Android_Projects/HenCoderPlus6-master/HenCoderPlus6-master/Arch/app/src/main/java/com/hencoder/arch/MainActivity.kt
package com.hencoder.arch

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

//M-VC
class MainActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    val data = DataCenter.getData()
    data1View.setText(data[0])
    data2View.setText(data[1])
  }
}